db/migrate/20150717135909_remove_unique_index_from_builds.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveUniqueIndexFromBuilds < ActiveRecord::Migration
+  def change
+    # Remove unique index on git_sha, users should be able to rebuild docker images against same SHA
```

Comment:
Ah, yea, I had thought that the .env file functionality could be part of the docker build stage. Where a .env file would be created and packaged as part of the image. 

Anyway, I'll re-add the unique index, and just re-push the docker image if the record already exists.